pepper-apis/src/main/java/org/broadinstitute/ddp/service/actvityinstancebuilder/ActivityInstanceFromDefinitionBuilder.java

```diff
@@ -204,6 +204,9 @@ public ActivityInstanceFromDefinitionBuilder buildFormInstance() {
                 var formInstance = context.getAIBuilderFactory().getFormInstanceCreator().createFormInstance(context);
                 context.setFormInstance(formInstance);
 
+                context.getAIBuilderFactory().getTemplateRenderHelper().addInstanceToRendererInitialContext(
+                        context, context.getFormInstance());
```

Comment:
I was thinking it'll be better to pass in the "form response" object instead. If we have the "form response", then I think we'll have less of an issue with render (and not needing to do 2 rendering runs) since the rendering will pull from the activity definition and template instead of the "form instance" label strings.

Reply:
Fixed: put form response and activity definition object to renderer context. 
Removed 2nd pass of templates apply.